x-pack/plugins/siem/server/lib/detection_engine/signals/filter_events_with_list.test.ts

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import uuid from 'uuid';
+import { filterEventsAgainstList } from './filter_events_with_list';
+import { mockLogger, repeatedSearchResultsWithSortId } from './__mocks__/es_results';
+
+import { ListClient } from '../../../../../lists/server';
+import { ListItemArraySchema } from '../../../../../lists/common/schemas';
+
+describe('filterEventsAgainstList', () => {
+  const someGuids = Array.from({ length: 13 }).map((x) => uuid.v4());
```

Comment:
nit: I'd just make a static list of uuid.v4()'s as an array of size 13 rather than generate them for each test run. If you get a failure it would help keep it deterministic.